src/components/ScaffoldTreeControls.vue

```diff
@@ -117,8 +120,8 @@ export default {
     // '__r/'
     findOrCreateRegion: function (data, paths, prefix) {
       //check if root region has been set
-      if (this.rootID === undefined && this.$module && this.$module.scene) {
-        this.treeData[0].id = this.$module.scene.getRootRegion().uuid;
+      if (this.rootID === undefined && this.module && this.module.scene) {
```

Comment:
Another question: Where does the `rootID` come from? I didn't find it in any other files.

Reply:
Remove that line please,

Reply:
You can safely remove this. Thanks.